src/test/adapter.ts

```diff
@@ -9,7 +9,7 @@ const assert = chai.assert;
 
 describe('PHP Debug Adapter', () => {
 
-    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject');
+    const TEST_PROJECT = path.normalize(__dirname + '/../../testproject/main');
```

Comment:
I'm fine without a test like that given that `tests/paths.ts` unit-tests the path mapping, but I don't understand why all these files have to be moved then if they are essentially not used at all.